todo_app/trello/trello_list.py

```diff
@@ -13,3 +13,6 @@ def add(self,trello_item):
     def trello_items(self):
         return self._items 
 
+    @property
+    def len(self):
+        return len(self._items)
```

Comment:
II have left this for now I have a feeling it may come in useful later.. thinking about it I could have added an exists call to enhance the test_client assert results..